src/cf-forms/src/cf-forms.less

```diff
@@ -10,43 +10,45 @@
 
 // Input borders
 @input-border:                  #5b616b;
-@input-border__hover:           @pacific;
-@input-border__focused:         @pacific;
-@input-border__active:          @pacific;
-@input-border__error:           @red;
-@input-border__warning:         @gold;
-@input-border__success:         @green;
+@input-border__hover:           #0072ce; // @pacific;
+@input-border__focused:         #0072ce; // @pacific;
+@input-border__active:          #0072ce; // @pacific;
+@input-border__error:           #d14124; // @red;
+@input-border__warning:         #ff9e1b; // @gold;
+@input-border__success:         #20aa3f; // @green;
 @input-border__selected:        #046b99; // $color-primary-alt-darkest
 
 // Input backgrounds
 @input-bg:                      #ffffff;
-@input-bg__active:              @pacific;
-@input-bg__disabled:            @gray-5;
+@input-bg__active:              #0072ce; // @pacific;
+@input-bg__disabled:            #f7f8f9; // @gray-5;
 
 // Input text
-@input-text__disabled:          @gray-80;
+@input-text__disabled:          #75787b; // @gray-80;
 @input-text__placeholder:       #aeb0b5;
 
 // Input icons
-@input-icon__error:             @red;
+@input-icon__error:             #d14124; // @red;
 
 // select
-@select-border:                 @gray-40;
-@select-icon:                   @gray-80;
-@select-icon-bg:                @gray-5;
+@select-border:                 #b4b5b6; // @gray-40;
+@select-icon:                   #75787b; // @gray-80;
+@select-icon-bg:                #f7f8f9; // @gray-5;
 
 // Large target area variables
 @input-lg-target:               #d6d7d9; // $color-gray-lighter
 @input-lg-target__selected:     #9bdaf1; // $color-primary-alt-light
 @input-lg-target__disabled:     #aeb0b5; // $color-gray-light
 
+@input-disabled:                #75787b; // @gray
 
 // Sizing variables
 
 @select-height:                  30px;
 
 // Import external dependencies
 
+@import (less) '../../cf-core/src/cf-base.less';
```

Comment:
Discussed in chat, let's figure out why `cf-core` isn't pulling this in correctly.